.github/workflows/create-project.yml

```diff
@@ -0,0 +1,36 @@
+name: "Create Project"
+
+on:
+  pull_request:
+  push:
+    branches:
+      - '[0-9]+.[0-9]+.x'
+      - 'refs/pull/*'
+    tags:
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        php_version:
+          - '@latest'
+          - '@lowest'
+    runs-on: ubuntu-latest
+    timeout-minutes: 3
+    steps:
+      - name: Checkout CI environment
+        uses: laminas/laminas-continuous-integration-action@v1
+        with:
+          php: ${{ matrix.php_version }}
+      - name: Checkout sourcecode
+        uses: actions/checkout@v3
```

Comment:
[laminas/laminas-ci-matrix-action](https://github.com/laminas/laminas-ci-matrix-action#actionscheckout-not-required) says <q>An actions/checkout step prior to this action is not required,</q> but requires an explicit checkout action to execute the command in the next step.